fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/commands/SharedCommand.scala

```diff
@@ -4,25 +4,19 @@ import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
-
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Promise
 import scala.sys.process._
 import scala.util.Failure
 import scala.util.Success
 import scala.util.Try
-
 import scala.meta.internal.fastpass.BuildInfo
 import scala.meta.internal.fastpass.FastpassEnrichments._
 import scala.meta.internal.fastpass.LogMessages
 import scala.meta.internal.fastpass.Time
 import scala.meta.internal.fastpass.Timer
-import scala.meta.internal.fastpass.pantsbuild.BloopPants
-import scala.meta.internal.fastpass.pantsbuild.Export
-import scala.meta.internal.fastpass.pantsbuild.IntelliJ
-import scala.meta.internal.fastpass.pantsbuild.MessageOnlyException
+import scala.meta.internal.fastpass.pantsbuild.{BloopPants, Export, IntelliJ, MessageOnlyException, PantsExportResult}
```

Comment:
I think this'll fail scalafmt / scalafix. You should be able to fix it from sbt by running `scalafmtAll` and `scalafixAll`.

Reply:
Yes, apparently that was the case.